src/Aspire.Hosting.Kafka/KafkaBuilderExtensions.cs

```diff
@@ -26,6 +26,7 @@ public static IResourceBuilder<KafkaServerResource> AddKafka(this IDistributedAp
         return builder.AddResource(kafka)
             .WithEndpoint(targetPort: KafkaBrokerPort, port: port, name: KafkaServerResource.PrimaryEndpointName)
             .WithImage("confluentinc/confluent-local", "7.6.0")
+            .WithImageRegistry("docker.io")
```

Comment:
Capture this in a class like the other ones?

Reply:
Done.